src/trezor-keyring.test.ts

```diff
@@ -15,16 +14,8 @@ import { Common, Chain, Hardfork } from '@ethereumjs/common';
 
 import { Address } from '@ethereumjs/util';
 import { SignTypedDataVersion } from '@metamask/eth-sig-util';
-import selfShim from '../test/self.shim';
-import navigatorShim from '../test/navigator.shim';
-import windowShim from '../test/window.shim';
-import { TrezorKeyring } from './trezor-keyring';
-
-global.window = windowShim;
-global.navigator = navigatorShim;
-global.self = selfShim;
```

Comment:
This shims are not needed for the tests, as we are creating stubs for `TrezorConnect`.